src/utils/validators/isValidNextHearingLocation.ts

```diff
@@ -0,0 +1,14 @@
+import OrganisationUnitApiResponse from "types/OrganisationUnitApiResponse"
+
+const isValidNextHearingLocation = (
```

Comment:
Does this need a test for completeness? Some of these validation functions have been a bit misleading in the past, test could help prevent that.

Reply:
Unit tests are added.